### ISSUE TITLE
Is gt really necessary in test.py

### ISSUE THREAD

**Issue by waaaaangchangwen** [KangLiao929/RecRecNet#1]
Hi, 
I'm trying to inference on my own image with your pre-trained checkpoint with test.py
And I found that gt images is part of the input but never used after turning it into numpy array
`gt_np = ((gt_tesnor[0]+1)*127.5).cpu().detach().numpy().transpose(1,2,0)`
Does that means I can put some dummy images in the gt folder just to make the code run without affecting checkpoint model's performance, or do I misunderstand anything
Thanks

**Comment by KangLiao929** [comment]
> Hi, I'm trying to inference on my own image with your pre-trained checkpoint with test.py And I found that gt images is part of the input but never used after turning it into numpy array `gt_np = ((gt_tesnor[0]+1)*127.5).cpu().detach().numpy().transpose(1,2,0)` Does that means I can put some dummy images in the gt folder just to make the code run without affecting checkpoint model's performance, or do I misunderstand anything Thanks

Hi,
The model doesn't require the gt images as input, they are only used to measure the rectangling performance (if you have one). Certainly, when you test your own images, you can just put some dummy images in the gt folder or just comment the codes for loading the gt images.
